### DISCUSSION TITLE
Code becomes RED within React Fragments(<> </>)

### DISCUSSION THREAD

**Discussion by hanancs** [vercel/next.js#25822] [Help] [answered]
While using short form of React fragments <> </> in JSX, code becomes Red on VS Code without any syntax error. When I wrap it within div tag, there's no such change of color. What could be reason behind this weird change? Pls can anybody help me figure out this??

![Capture](https://user-images.githubusercontent.com/50701475/120916571-01bf3880-c6c8-11eb-857c-a237432c1ae5.PNG)

**Comment by imteekay** [comment, marked as answer]
Not sure, but it seems to be related to your vscode theme, not NextJS. Tested it in a production app I'm working on and it looks ok for me.

![Screenshot from 2021-06-06 12-32-34](https://user-images.githubusercontent.com/5835798/120930366-5dd38e00-c6c3-11eb-8783-5dc6083e51f3.png)

**Reply by leerob** [reply]
Correct!

**Reply by hanancs** [reply]
Thanks for answering me @leandrotk & thanks for confirming @leerob 

I finally came to know the fact that the change of color is because of VS code extension Babel ES6 / ES7.

![Capture](https://user-images.githubusercontent.com/50701475/121525294-c8633180-ca15-11eb-8ec2-bf1c4c94a168.PNG)

**Reply by dhruvil444** [reply]
Works Good Now Thanks for Solution

**Reply by Meron-Kipgen** [reply]
i delete babel and now it work, thanks
